packages/@aws-cdk/aws-appsync/test/verify.integ.graphql-s3.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+function error {
+  printf "\e[91;5;81m$@\e[0m\n"
```

Comment:
curious: what's happening here?

Reply:
It helps format the error message. I got it from the buildup/builddown script

Reply:
Basically, the way it works is the \e specifies an escape sequence.

So bascially this function has three components:
- `\e[91;5;81m$@` - formatting the input text
- `\e[0m` - remove all formatting attributes (sanity for future texts)
- `\n` - newline

Breaking down the first one `\e[91;5;81m$@`:
- `91` - the color red
- `5`- blinking 
- `81` - honestly I have no idea what this is for.. removing it doesn't change anything for me
- `$@` - input to function